Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -419,6 +419,17 @@ def check_for_mirrors(formula, old_mirrors, new_mirrors, args:)
     end
   end
 
+  def update_url(old_url, old_version, new_version)
+    new_url = old_url.gsub(old_version, new_version)
+    return new_url if (old_version_split = old_version.split(".")).length < 2
+    return new_url if (new_version_split = new_version.split(".")).length < 2
+    return new_url if old_version_split.length != new_version_split.length
```

Comment:
I can probably squash this into a single check since we already confirmed min length
```suggestion
    return new_url if (new_version_split = new_version.split(".")).length != old_version_split.length
```